context/generated/teamcity-agent-EAP-linux-sudo.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 cd ../..
-docker pull ubuntu:20.04
+docker pull ubuntu:22.04
 echo TeamCity/webapps > context/.dockerignore
 echo TeamCity/devPackage >> context/.dockerignore
 echo TeamCity/lib >> context/.dockerignore
-docker build -f "context/generated/linux/MinimalAgent/Ubuntu/20.04/Dockerfile" -t teamcity-minimal-agent:EAP-linux "context"
+docker build -f "context/generated/linux/MinimalAgent/Ubuntu/22.04/Dockerfile" -t teamcity-minimal-agent:EAP-linux "context"
 echo 2> context/.dockerignore
 echo TeamCity >> context/.dockerignore
-docker build -f "context/generated/linux/Agent/Ubuntu/20.04/Dockerfile" -t teamcity-agent:EAP-linux "context"
+docker build -f "context/generated/linux/Agent/Ubuntu/22.04/Dockerfile" -t teamcity-agent:EAP-linux "context"
 docker build -f "context/generated/linux/Agent/Ubuntu/20.04-sudo/Dockerfile" -t teamcity-agent:EAP-linux-sudo "context"
```

Comment:
Hi !

The line for 22.04-sudo may have been forget here ! :)

Reply:
Hi @arsiesys !
Good catch, thank you very much!
The scripts, along with several hyperlinks in the markdown, have been updated to reflect the correct base image or Dockerfile link, pointing to the newer version.